README.md

```diff
@@ -1,22 +1,20 @@
-# Pana Backend Code Challenge
+# Coupa Travel Backend Code Challenge
 
 ## The Brief
 
-Pana contracts with a number of different businesses (_businesses_ table) to supply their travel needs.  Each of those businesses have a number of employees (_employees_ table) which can access Pana in order to schedule, book, and modify their upcoming trips. However, each business has unique information they want to store about individual employees in Pana.  This information will help them track their travel and do month end reconciliation. 
+Coupa Travel contracts with a number of different businesses (_businesses_ table) to supply their travel needs.  Each of those businesses have a number of employees (_employees_ table) which can access Coupa Travel in order to schedule, book, and modify their upcoming trips. However, each business has unique information they want to store about individual employees in Coupa Travel.  This information will help them track their travel and do month end reconciliation. 
 
 For example:
 
-ABC Corp wants to store custom fields _Employee ID_ and _Cost Center_ with each of their employees. (The employee id is ABC's internal id, not Pana's).
+ABC Corp wants to store custom fields _Employee ID_ and _Cost Center_ with each of their employees. (The employee id is ABC's internal id, not Coupa Travel's).
 
 Dunder Mifflin wants to store custom fields _Sales Ranking_ and _Michael Scott Rating_. 
 
 Other businesses may want more custom fields, other companies may want fewer.  There may be between 0 to N custom fields per business. The solution you choose should support these other businesses' custom fields without requiring additional engineering work.
 
 ## The Challenge
 
-Using the included API server and your code wizardry, the goal is to implement functionality which allows custom data to be associated with a business' travel employees that are stored in the database.
-
-The goal of this task is to build an understanding of how you would structure databases and APIs, so no coding is actually necessary. We will use `NOTES.md` to write out your thoughts. You can also use DbDiagram https://dbdiagram.io/home (you can create a diagram without signing in) or another visualization tool if you would find that helpful, but it is not required.
+The goal of this task is to build an understanding of how you would structure databases and APIs, so no coding is actually necessary. We will use `NOTES.md` to write out your thoughts. You can also use DbDiagram https://dbdiagram.io/home (you can create a diagram without signing in and we provide the starter schema in starterSchema.txt) or another visualization tool if you would find that helpful, but it is not required.
```

Comment:
we should update that "We will use `NOTES.md` to write out your thoughts." to something like "You can screen share a text editor of your choice to write out your thoughts." Essentially what we end up telling them. Notes.md is never actually used

Reply:
I also think this is the cut off point in terms of what we should prep the candidate with. It might be nice to add some sort of big header, or line below this to indicate to the candidate where new info starts when they jump in. We also might want to add a small note that says something like "requirements and more details will be provided at the start of the interview below"

Reply:
Oh we should also mention that its preferred to do this with SQL, but if No-SQL is preferred, then be prepared to justify their approach

Reply:
I'm a little confused here. So are we removing the Core Requirements section and giving them the challenge during the interview?

Reply:
I was intending on copying the "public" portion somewhere else, and keeping this repo as what to show during the live interview

Reply:
I'm not sure I understand, I'm gonna merge this PR since we have an interview this afternoon, then let's open another PR with these changes, maybe we can store it on another branch or something